crates/services/upgradable-executor/wasm-executor/src/ext.rs

```diff
@@ -84,7 +84,7 @@ mod host {
         pub(crate) fn peek_next_txs_size(
             gas_limit: u64,
             tx_count_limit: u32,
-            size_limit: u32,
+            block_transaction_size_limit: u64,
```

Comment:
It is a breaking change

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2195/commits/e748837be51c4d738912768c0aa33d605444118f